website_compassion/forms/partner_forms.py

```diff
@@ -50,18 +52,34 @@ class PartnerCoordinatesForm(models.AbstractModel):
         "title",
         "firstname",
         "lastname",
+        "name",
         "street",
         "zip",
         "city",
         "country_id",
         "email",
     ]
 
+    def _form_write(self, values):
+        """ Nothing to do on write, we handle everything in other methods.
+        """
+        return self.main_object.sudo().write(values.copy())
+
```

Comment:
Wrong comment, plus we don't want to write as sudo otherwise we lose the info that the sponsor made himself the changes. It's better to add missing security rules if there are any (we already added some in production).
```suggestion
```